kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/ProductOrderService.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.good.patterns.challenges;
+
+public class ProductOrderService {
```

Comment:
I know that name of this class was suggested in the task, but for me it will be difficult to find out which class I should use (SalesService vs ProductOrderService) without checking implementation. If service uses other services then it can be called Facade, or Processor - to somehow highlight that it provides high-level functionality.